src/core/lv_obj_property.h

```diff
@@ -83,12 +88,28 @@ struct _lv_property_name_t {
 typedef struct {
     lv_prop_id_t id;
     union {
-        int32_t num;                /**< Number integer number (opacity, enums, booleans or "normal" numbers)*/
-        bool enable;                /**< booleans*/
-        const void * ptr;           /**< Constant pointers  (font, cone text, etc)*/
-        lv_color_t color;           /**< Colors*/
-        lv_value_precise_t precise; /**< float or int for precise value*/
-        lv_point_t point;           /**< Point*/
+        struct {
+            union {
```

Comment:
num + num_2 don't feels that means two properties with two values, not a blocker but for clarification why not suggest something like C++ does for their map structure by prepending first_ and second_ prefixes for the properties?

for example we would have first_num, and second_num , which also would enforce the correct order of filling the properties.

Reply:
cc @XuNeo

Reply:
I makes sense. Instead of using first and second, I simply put arg1 and arg2 so when two args are needed, it's specified explicitly.

E.g.
```
    prop.id = LV_PROPERTY_ROLLER_OPTIONS;
    prop.arg1.ptr = "One\nTwo\nThree";
    prop.arg2.num = LV_ROLLER_MODE_NORMAL;
    lv_obj_set_property(obj, &prop);
```